buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -170,7 +170,13 @@ timeout(time: 12, unit: 'HOURS') {
                             ADOPTOPENJDK_BRANCH = ADOPTOPENJDK_MAP.get(SDK_VERSION).get('branch')
                         }
 
-                        builds["build_test_JDK${SDK_VERSION}_${SPEC}"] = { build(REPO, BRANCH, SHAS, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, SPEC, SDK_VERSION, BUILD_NODE, TEST_NODE, EXTRA_GETSOURCE_OPTIONS, EXTRA_CONFIGURE_OPTIONS, EXTRA_MAKE_OPTIONS, OPENJDK_CLONE_DIR, ADOPTOPENJDK_REPO, ADOPTOPENJDK_BRANCH) }
+                        def job_name = "Pipeline-Build-Test-JDK${SDK_VERSION}-${SPEC}"
+                        def AUTOMATIC_GENERATION = params.AUTOMATIC_GENERATION
```

Comment:
Might as well move this def into the section at line 93 and initialize the same way.